src/modules/LanItems.rb

```diff
@@ -2249,11 +2253,14 @@ def ProposeItem
       @ipaddr = ""
       @netmask = ""
       @bootproto = "dhcp"
+# storage-ng
+=begin
       # #176804
       if Storage.IsDeviceOnNetwork(NetworkStorage.getDevice("/")) != :no
         @startmode = "nfsroot"
         Builtins.y2milestone("startmode nfsroot")
       end
+=end
```

Comment:
Ok, so it looks like we forgot about this file when bringing the functionality back. So this is ok from the merge POV... but we need to fix it ASAP because it looks like https://trello.com/c/qlK9iJpH/1031-3-new-storage-network-based-root-check-readded is not as done as we though.